src/components/cycle-section/learning-units-section/learning-units-list/LearningUnitsList.js

```diff
@@ -14,7 +14,13 @@ const LearningUnitsList = ({ learningUnits }) => {
   };
 
   const renderListItem = (unit) => {
-    return <LearningUnitListItem unit={unit} showSuccess={showSuccess} />;
+    return (
+      <LearningUnitListItem
+        unit={unit}
```

Comment:
This should be called learningUnit instead of unit 🙃 
I know its inherited, feel free to omit it if you want to

Reply:
(everywhere, not just in this component)

Reply:
Done, it was hurting my eyes too